File: MT.Model/Business/Salary.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Salary:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Salary:EntityBase.
    /// </summary>
    public class Salary : EntityBase
    {
        [Key]
        public Guid SalaryID { get; set; }
        public string SalaryCode { get; set; }
        public string SalaryName { get; set; }
        public int Month { get; set; }
        public int Year { get; set; }
        public int Money { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }

    }
    #endregion
}

File: BECore/app/model/Salary.js
/// <summary>
/// Model EXTJS Salary.
/// </summary>
Ext.define('QLDT.model.Salary', {
    extend: 'Ext.data.Model',
    idProperty: 'SalaryID',
    idType: 'Guid',
    fields: [
		{ name: "SalaryID", type: "string" },
		{ name: "SalaryCode", type: "string" },
		{ name: "SalaryName", type: "string" },
		{ name: "Month", type: "int" },
		{ name: "Year", type: "int" },
		{ name: "Money", type: "int" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
    ]
});

